controls/datagrid/columns/overview.md

```diff
@@ -1,77 +1,49 @@
 ---
-title: Overview
-page_title: .NET MAUI DataGrid Documentation - Columns Overview
+title: Defining Columns
+page_title: .NET MAUI DataGrid Documentation - Defining Columns
 description: Check our &quot;Overview&quot; documentation article for Telerik DataGrid for .NET MAUI.
 position: 0
 previous_url: /controls/datagrid/columns/datagrid-columns-overview
 slug: datagrid-columns-overview
 ---
 
-# .NET MAUI DataGrid Columns
+# .NET MAUI DataGrid Defining Columns
 
-You can add columns in your `RadDataGrid` by working with the Columns collection of the control. It has three approaches that you can take to define different columns:
+The DataGrid for .NET MAUI provides three approaches that you can take to define different columns:
 
-* `Manually`: by adding columns to the `RadDataGrid.Columns` collection
-* `Automatically`: by setting `RadDataGrid.AutoGenerateColumns="True"`
-* `Mixed`: by adding columns to the `RadDataGrid.Columns` collection and also set `RadDataGrid.AutoGenerateColumns="True"`
+* `Manually`: by adding columns to the DataGrid's `Columns` collection and set `AutoGenerateColumns` property to `False`.
```

Comment:
* Manually, Automatically, and Mixed do not seem to be elements of the code, so they must use the regular font instead of `monospace`.
* Move Automatically before Manually to match the order of the headings.